site/static/diagrams/gameserver-states.dot

```diff
@@ -23,9 +24,14 @@ digraph {
     PortAllocation -> Creating [ label="allocated unused port", color="blue" ]
     Creating -> Starting [ label="created pod", color="blue" ]
     Starting -> Scheduled [ label="we have a pod, fetch its address", color="blue" ]
+    Scheduled -> Reserved [ label="SDK.reserved(seconds)", color="purple" ]
     Scheduled -> RequestReady [ label="SDK.ready()", color="purple" ]
     RequestReady -> Ready [ label="ready to be allocated", color="blue" ]
+    Reserved -> Ready [label="if seconds > 0  \nand failed to call  \nSDK.allocate()  " color="purple"]
```

Comment:
It's difficult to tell from the diagram which arrow this note belongs to. Maybe there isn't any avoiding that though...

Reply:
Tweaked some alignment, think it looks a bit better. PTAL.